### PR TITLE
Add documentation for iomete adapter

### PR DESCRIPTION
**Pull request by namigaliyev**
## Description & motivation

Added iomete adapter. iomete is an all-in-one data infrastructure platform.

Related to [dbt-iomete](https://github.com/iomete/dbt-iomete)

## Checklist
If you added new pages (delete if not applicable):
- [x] The page has been added to `website/sidebars.js`
- [x] The new page has a unique filename

### CODE REVIEW

File: website/docs/reference/warehouse-profiles/iomete-profile.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+title: "Apache iomete Profile"
+---
+
+## Overview of dbt-iomete
+
+**Maintained by:** iomete
+**Author:** Namig Aliyev
+**Source:** [Github](https://github.com/iomete/dbt-iomete)
+**dbt Cloud:** Currently un-supported
+**dbt Slack channel** [Link to channel]()
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/iomete-profile.md]: if you want a Slack channel, please request one in the [community Slack](https://www.getdbt.com/community/join-the-community/)'s [#channel-requests channel](https://getdbt.slack.com/archives/C01D8J8AJDA)!

File: website/docs/reference/warehouse-profiles/iomete-profile.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Apache iomete Profile"
```
Comment (dataders) [inline website/docs/reference/warehouse-profiles/iomete-profile.md]: nit: is iomete an official Apache project? I know Iceberg and Spark are, but this combo isn't?
```suggestion
title: "iomete Profile"
```

**Review by dataders — CHANGES_REQUESTED** [review]
thanks @namigalyv! We're excited to have this listed on the doc.

Some changes:
1. can you link to imomete's docs page somewhere in this repo?
2. Can you please also provide guidance on how to set iomete-specific settings using dbt? Our docs's new guide [Documenting an Adapter: Topics and Pages to Cover](https://docs.getdbt.com/docs/contributing/documenting-a-new-adapter#topics-and-pages-to-cover) gives a good overview.
3. I opened https://github.com/iomete/dbt-iomete/pull/1 to show how much more simple the adapter might get. I could even see iomete working inside of dbt-spark completely!
LMK if you have additional questions!

File: website/docs/reference/warehouse-profiles/iomete-profile.md (hunk 1 above)
Comment (namigaliyev) [reply website/docs/reference/warehouse-profiles/iomete-profile.md]: I already mentioned about to create a dedicated channel for dbt-iomete in #channel-request.

**Comment by namigaliyev** [comment]
> thanks @namigalyv! We're excited to have this listed on the doc.
> 
> Some changes:
> 
> 1. can you link to imomete's docs page somewhere in this repo?
> 2. Can you please also provide guidance on how to set iomete-specific settings using dbt? Our docs's new guide [Documenting an Adapter: Topics and Pages to Cover](https://docs.getdbt.com/docs/contributing/documenting-a-new-adapter#topics-and-pages-to-cover) gives a good overview.
> 3. I opened [iomete/dbt-iomete#1](https://github.com/iomete/dbt-iomete/pull/1) to show how much more simple the adapter might get. I could even see iomete working inside of dbt-spark completely!
>    LMK if you have additional questions!

Hi, 

1. iomete dbt documentation link added.
2. All iomete specific settings already documented on iomete-profile.md

**Comment by dataders** [comment]
> 2\. All iomete specific settings already documented on iomete-profile.md

what about some features like indexes and "unlogged" tables? Here's today's versions of [dbt-postgres configs](https://docs.getdbt.com/reference/resource-configs/postgres-configs) and [dbt-greenplum's configs](https://docs.getdbt.com/reference/resource-configs/greenplum-configs) (which is also somewhat postgres-based)

**Comment by namigaliyev** [comment]
> > 2. All iomete specific settings already documented on iomete-profile.md
> 
> what about some features like indexes and "unlogged" tables? Here's today's versions of [dbt-postgres configs](https://docs.getdbt.com/reference/resource-configs/postgres-configs) and [dbt-greenplum's configs](https://docs.getdbt.com/reference/resource-configs/greenplum-configs) (which is also somewhat postgres-based)

Currently, no additional configuration is required to work with dbt-iomete. We've fixed the first Iceberg-specific issue and added it to the documentation. We will update the documentation if different situations arise after we start working with dbt.

**Review by dataders — APPROVED** [review]
thanks @namigalyv!

**Pull request merged**